src/plugins/intel_cpu/src/transformations/cpu_opset/common/pass/stateful_sdpa_fusion.cpp

```diff
@@ -258,6 +258,27 @@ StatefulSDPAFusion::StatefulSDPAFusion() {
         else
             assign_v_node->set_arguments({new_node->output(2)});
 
+        // Markup pattern: ReadValue->ScaledDotProductAttentionWithKVCache->Assigin, ReadValue can't repalce to
+        // ReadValueWithSubgraph in this pattern.
```

Comment:
```suggestion
        // Markup pattern: ReadValue->ScaledDotProductAttentionWithKVCache->Assigin, so that ReadValue can't be replaced with
        // ReadValueWithSubgraph in this pattern.
```